package.json

```diff
@@ -17,9 +17,11 @@
     "eslint-config-crockford": "2.0.0",
     "eslint-config-standard": "14.1.1",
     "eslint-plugin-import": "2.22.0",
+    "eslint-plugin-jest": "^24.1.3",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-promise": "4.2.1",
     "eslint-plugin-standard": "4.0.1",
+    "jest": "26.6.3",
```

Comment:
jest selbst muss dafür installiert sein? schräg

Reply:
alternative waere, jest version in der eslintrc vorzuschreiben, das waere die alternative.